proxy/proxy.go

```diff
@@ -51,19 +61,41 @@ func main() {
 
 	loggedRouter := handlers.LoggingHandler(os.Stdout, proxyRouter)
 
-	err = http.ListenAndServe(":"+c.Port, loggedRouter)
+	err = http.ListenAndServe(":" + c.Port, loggedRouter)
 	if err != nil {
 		fmt.Println(err)
 	}
 }
-
+func CallbackUrl() (string, error) {
+	var vcapApp VcapApplication
+	err := json.Unmarshal([]byte(c.VcapAppRaw), &vcapApp)
+	if err != nil {
+		return "", err
+	}
+	// we try first ssl to see if it can be used
```

Comment:
Yes it can be, or something common like FORCE_SSL. Here I did it this way to always force ssl when it's available. It's more cause of my conviction to always have web app secured by ssl. Here in add, I feel like it's really sensitive to go through http instead of https.

Maybe an environment variable disable this check can be used ?